gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/pages/BasePage.html

```diff
@@ -10,41 +10,41 @@
 		<ul class="navIntraTool actionToolbar" wicket:id="gradebookPageNav">
 			<li class="firstToolBarItem">
 				<span>
-					<a wicket:id="gradebookPageLink" wicket:message="title:link.gradebook.tooltip">
-						<wicket:message key="link.gradebook" />
+					<a wicket:id="gradebookPageLink">
 						<span wicket:id="screenreaderlabel" class="sr-only"></span>
+						<wicket:message key="link.gradebook" />
```

Comment:
doesn't this message need to go inside the spans?

Reply:
i don't understand. I have only removed `wicket:message="title:link.quickentry.tooltip` because it was reading twice for screen readers.

Reply:
ok got it.